src/Application.cpp

```diff
@@ -15,21 +16,14 @@ namespace ccm
 		}
 
 		/* Create window */
-		m_window = SDL_CreateWindow("Game", SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED, 1280, 720, SDL_WINDOW_RESIZABLE);
+		m_window = SDL_CreateWindow(windowName.c_str(), SDL_WINDOWPOS_CENTERED,
```

Comment:
Actually, since we're only forwarding the raw pointer anyway, we can pass `windowName` as a std::string_view instead, to save on unnecessary allocations.